test/tarantool/vinyl_test.lua

```diff
@@ -0,0 +1,31 @@
+require('strict').on()
+
+local t = require('luatest')
+local g = t.group()
+
+local metrics = require('metrics')
+local fun = require('fun')
+
+g.after_each(function()
+    -- Delete all collectors and global labels
+    metrics.clear()
+end)
+
+g.before_all(function()
+    local s_vinyl = box.schema.space.create(
+        'test_space',
+        {if_not_exists = true, engine = 'vinyl'})
+    s_vinyl:create_index('pk', {if_not_exists = true})
+    metrics.clear()
+
+    -- Enable default metrics collections
+    metrics.enable_default_metrics()
+end)
+
+g.test_vinyl_metrics_present = function()
```

Comment:
Is there any cases when vinyl metrics are not present? If it is then it should be covered too.

Reply:
Vinyl metrics is always present